package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "3.4.5",
+  "version": "3.4.6",
```

Comment:
Since you're adding a new, significant feature (a new filing type), how about updating the minor release number?